java-lib/src/main/avro/Reporting.avdl

```diff
@@ -110,4 +110,22 @@ protocol Reporting {
     union {null, array<string>} tags = null;
     union {null, map<array<string>>} dimensions = null;
   }
+
+  record ReportMetric {
+    string metric;
+    long timestamp;
+    double value;
+    string host = "unknown";
+    string table = "tsdb";
+    map<string> annotations = {};
```

Comment:
Ultimately to support multi-valued keys, we need to use a class unfortunately, this is the chance to pull it off

Reply:
done